pom.xml

```diff
@@ -87,6 +88,32 @@ under the License.
     </dependencies>
 
     <profiles>
+        <profile>
+            <id>java-11</id>
+            <activation>
+                <jdk>[11,17)</jdk>
+            </activation>
+            <properties>
+                <target.java.version>11</target.java.version>
+                <maven.compiler.source>11</maven.compiler.source>
+                <maven.compiler.target>11</maven.compiler.target>
+                <mcp.enabled>false</mcp.enabled>
+            </properties>
+        </profile>
+
+        <profile>
+            <id>java-17-plus</id>
```

Comment:
I think `java-17` should be enough. Otherwise, when we add the profile for java21, we need to update the id of this profile as well.